dev/DynamicDependency/MsixDynamicDependency.cpp

```diff
@@ -19,6 +19,15 @@ bool IsStaticPackageGraphEmpty()
     (void) LOG_HR_IF_MSG(HRESULT_FROM_WIN32(rc), (rc != APPMODEL_ERROR_NO_PACKAGE) && (rc != ERROR_INSUFFICIENT_BUFFER), "GetCurrentStaticPackageInfo rc=%d", rc);
     return rc == APPMODEL_ERROR_NO_PACKAGE;
 }
+
+// Temporary check to prevent accidental misuse and false bug reports until we address Issue #567 https://github.com/microsoft/ProjectReunion/issues/567
+bool IsElevated(HANDLE token = nullptr)
+{
+    wistd::unique_ptr<TOKEN_MANDATORY_LABEL> tokenMandatoryLabel;
+    FAIL_FAST_IF_FAILED(wil::get_token_information_nothrow(tokenMandatoryLabel, !token ? GetCurrentThreadEffectiveToken() : token));
```

Comment:
Oh also, use `wil::get_token_information_failfast` and let it deal with it.

Reply:
Forgot about that one. Thanks!